.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -5,33 +5,7 @@ about: Have a feature you'd like to see in the functions SDK? Request it through
 title: ''
 labels: ''
 assignees: ''
-
 ---
 
-<!--
-Thank you for contributing to the Firebase community!
-
-Have a feature request?
-========================
 Great, we love hearing how we can improve our products! However, GitHub is not the place to submit them. Please submit your feature requests to:
 https://firebase.google.com/support/contact/bugs-features/
-
-Have a usage question?
-=======================
-We get lots of those and we love helping you, but GitHub is not the best place for them and they
-will be closed. Here are some resources to get help:
-
-- Start with the quickstart: https://firebase.google.com/docs/functions/write-firebase-functions
```

Comment:
Should we move these to the readme then?

Reply:
Sounds good